clang/include/clang/AST/Decl.h

```diff
@@ -2005,12 +2019,12 @@ class FunctionDecl : public DeclaratorDecl,
   ParmVarDecl **ParamInfo = nullptr;
 
   /// The active member of this union is determined by
-  /// FunctionDeclBits.HasDefaultedFunctionInfo.
+  /// FunctionDeclBits.HasExtraFunctionInfo.
   union {
     /// The body of the function.
     LazyDeclStmtPtr Body;
     /// Information about a future defaulted function definition.
-    DefaultedFunctionInfo *DefaultedInfo;
+    ExtraFunctionInfo *ExtraInfo;
```

Comment:
I prefered the old name. `DefaultedOrDeletedInfo` if you really want. `Extra` is pretty vacuous.

Reply:
Agreed 100%

Reply:
Yeah, I didn’t just want to keep it as `DefaultedFunctionInfo` because it’s not just for defaulted functions anymore; `ExtraFunctionInfo` is the best I could come up w/ because I’m bad at naming things... I like `DefaultedOrDeletedInfo`, so I’ll change it to that if there aren’t any objections.